src/storage/storage.cpp

```diff
@@ -312,7 +312,7 @@ std::vector<std::pair<bool, boost::filesystem::path>> Storage::GetStaticFiles()
     {
         if (file.first == REQUIRED && !boost::filesystem::exists(file.second))
         {
-            throw util::exception("Could not find required filed: " + std::get<1>(file).string());
+            throw util::exception("Could not find required file(s): " + std::get<1>(file).string());
```

Comment:
Does this exception can happen for several files?

Reply:
It looks like it fires on a per-file basis, so we may not need the plural, but it could potentially be multiple files that it can't find